modules/tests/photons_control_tests/test_multizone.py

```diff
@@ -780,7 +780,9 @@ def compare_received(self, by_light):
             )
 
         async it "has options", sender:
-            msg = SetZonesEffect("move", speed=5, duration=10, power_on_duration=20)
+            msg = SetZonesEffect(
+                "move", speed=5, duration=10, direction="RIGHT", power_on_duration=20
```

Comment:
```suggestion
                "move", speed=5, duration=10, direction="LEFT", power_on_duration=20
```

and then change the test to show it made a difference